test/com/cloudogu/ces/cesbuildlib/SCMManagerTest.groovy

```diff
@@ -0,0 +1,188 @@
+package com.cloudogu.ces.cesbuildlib
+
+import org.junit.Before
+import org.junit.Test
+
+import static org.assertj.core.api.Assertions.assertThat 
+
+class SCMManagerTest {
+
+  ScriptMock scriptMock = new ScriptMock()
+  SCMManager scmm = new SCMManager(scriptMock, "credentialsID")
+
+  @Before
+  void init() {
+    scmm.repositoryUrl = "scm/repo"
+    scriptMock.env.put("GIT_USER", "user")
+    scriptMock.env.put("GIT_PASSWORD", "pw")
+  }
+
+  @Test
+  void "getting all pull requests"() {
+    String response = """HTTP/2 200
+some: header
```

Comment:
I find these multi line strings generally hard to read and maintain. We could think about using `JsonOutput`, just like in  `SCMManager`.

Also, now that we have the `HttpClient`, we could simplify these tests by making the client return a mock body and httpCode, instead of using "real" curl output here.

Reply:
Fixed in 2d6a8c4b